test/test_jit.py

```diff
@@ -287,7 +287,7 @@ def f(x, y):
             return z1 * z2
         x = Variable(torch.randn(4, 4).float().cuda())
         y = Variable(torch.randn(4, 4).float().cuda())
-        trace, _ = torch.jit.trace(f, (x, y), nderivs=0)
+        trace, _ = torch.jit.create_trace(f, (x, y), nderivs=0)
```

Comment:
Maybe we should call it `get_trace_graph` or sth? Just to make it clear it will give you a graph, and uses tracing, but is not connected to `trace` in any way

Reply:
sounds good.